submissions/NJul/a-tiny-JS-world-oop/index.js

```diff
@@ -0,0 +1,91 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/NJul/a-tiny-JS-world
+   Web app: https://njul.github.io/a-tiny-JS-world/
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this.friends = Array.isArray(friends) ? friends.join(', ') : 'no one';
```

Comment:
What if want to add a friend or two later? If this prop is stored as a string then method of adding would be much different from a method of assigning initial set of friends. Not really consistent approach.